gr-audio/lib/osx/osx_sink.cc

```diff
@@ -933,7 +933,9 @@ OSStatus osx_sink::au_output_callback(void* in_ref_con,
                     << "number of available items changing "
                     << "unexpectedly (should never happen): was " << in_number_frames
                     << " now " << t_n_output_items;
-                GR_LOG_ERROR(d_logger, msg.str());
+                // commented due to error "invalid use of member 'd_logger' in static
+                // member function"
+                // GR_LOG_ERROR(d_logger, msg.str());
```

Comment:
Instead of the comment, can you encapsulate these lines into 
```cpp
#if _OSX_AU_DEBUG_RENDER_
// ...
#endif
```

Now, frankly, that doesn't fix the fact that we're calling `d_logger` in a static function, but your goal was to make it compile (and that'll do that) while also maintaining consistency. Maybe someone else will fix the issue that arises if _OSX_AU_DEBUG_RENDER_ is actually asserted (maybe you? :innocent: ) But for now, that would do.